cmd/copy.go

```diff
@@ -360,9 +360,9 @@ func (raw rawCopyCmdArgs) cook() (CookedCopyCmdArgs, error) {
 	}
 
 	// If the given blobType is AppendBlob, block-size-mb should not be greater than
-	// 4MB.
+	// 100MB.
 	if cookedSize, _ := blockSizeInBytes(raw.blockSizeMB); cooked.blobType == common.EBlobType.AppendBlob() && cookedSize > common.MaxAppendBlobBlockSize {
-		return cooked, fmt.Errorf("block size cannot be greater than 4MB for AppendBlob blob type")
+		return cooked, fmt.Errorf("block size cannot be greater than 100MB for AppendBlob blob type")
```

Comment:
let's try and pull the constant here in the error message so we don't have to change this text every time

Reply:
If SDK has any constants defined, better to use that so that with upgrade in SDK we always reflect back to the latest limit.

Reply:
I completely agree here but the constant values to 104857600 which is unreadable imo. I can still make this change if the rest of the team is aligned with this.

Reply:
@vibhansa-msft , please see my comment here (https://github.com/Azure/azure-storage-azcopy/pull/2480#discussion_r1420758221). 

I was unable to find the azblob SDK constant that would contain MaxAppendBlobBlock. Here is the Golang SDK PR: https://github.com/Azure/azure-sdk-for-go/pull/21424 and the .NET PR: https://github.com/Azure/azure-sdk-for-net/pull/34548 in regards to high throughput AppendBlob. 

Otherwise I've updated this error message to pull the constant in AzCopy directly.

Reply:
if these constants don't already exist, we should add some to the GO SDK. @vibhansa-msft Could you add this item to the Go SDK backlog?

Reply:
I started an email thread in regard to this ("RE: High Throughput Append Blob Constant"). I will resolve this conversation as this is not blocking this PR.